platform/security/security-review-process.md

```diff
@@ -0,0 +1,25 @@
+# Security review process
+
+## Context
+
+Before launching an application on VA.gov, the feature should be reviewed by a VSP security engineer.
```

Comment:
Are we clear who we'd call a "security engineer" ? I feel like it would be an exaggeration to call myself one.

Reply:
right now I consider that list to be [gunsch], but I think we're hoping to eventually hire engineers with security background